src/sentry/api/serializers/models/project.py

```diff
@@ -955,6 +955,7 @@ def serialize(
                     "sentry:highlight_context",
                     attrs["highlight_preset"].get("context", {}),
                 ),
+                "highlightPreset": attrs["highlight_preset"],
```

Comment:
By adding it here, anytime a user reads their highlights, that same payload will also contain the defaults. We can let the frontend sort out resubmitting a PUT request with the preset, since the API is the exact same.